README.rst

```diff
@@ -293,3 +293,26 @@ Sentry:
     }
 
 You could also log to a file by using the built-in ``logging.FileHandler``.
+
+Useful Recipes
+==============
+These recipes aren't officially supported features of `django-postgres-queue`. We provide them so that you can mimick some of the common features in other task queues.
+
+`ALWAYS_EAGER`
+--------------
+Celery uses the `ALWAYS_EAGER` setting to run a task immediately, without queueing it for a worker. It could be used during tests, and while debugging in a development environment with any workers turned off.
+
+.. code:: python
+
+    class EagerAtLeastOnceQueue(AtLeastOnceQueue):
+        def enqueue(self, *args, **kwargs):
+            job = super().enqueue(*args, **kwargs)
+            if settings.CELERY_ALWAYS_EAGER:
```

Comment:
Let's use a different setting like `QUEUE_ALWAYS_EAGER`

Reply:
I originally had DPQ_ALWAYS_EAGER, but that looked like we supported it. CELERY_ALWAYS_EAGER made it clearer that it wasn't our thing, it was their thing we are mimicking. What benefit does QUEUE_ALWAYS_EAGER give? Who is the QUEUE namespace?

Reply:
It's not a namespace it's just saying what is always eager